Kiss.Bff.EndToEndTest/AnonymousContactmomentZaak/AnonymousContactmomentZaakScenarios.cs

```diff
@@ -65,7 +65,7 @@ public async Task RegisterContactmomentBijZaakI()
 
             await Step("And a checked checkbox is visible");
 
-            await Expect(Page.GetByRole(AriaRole.Checkbox, new() { Name = "ZAAK-2023-001" })).ToBeCheckedAsync();
+            await Expect(Page.GetByRole(AriaRole.Radio, new() { Name = "ZAAK-2023-001 (Zaaktype:" })).ToBeCheckedAsync();
```

Comment:
are you sure this is correct? 
Name = "ZAAK-2023-001 (Zaaktype:"
or did you accidentally pasted in '(Zaaktype:'